pkg/handlers/valkey/valkey_test.go

```diff
@@ -266,15 +278,11 @@ var _ = Describe("valkey.Handler", func() {
 				defaultServiceManagerMock(data)
 				mocks.serviceUserManager.On("Get", mock.Anything, data.username, projectName, data.serviceName, mock.Anything).
 					Return(nil, aiven.Error{
-						Message: "Service user does not exist",
-						Status:  404,
+						Message:  "aiven-error",
```

Comment:
Er dette fordi dere har funnet ut at det er dette man får fra API'et?

Reply:
Bra sett, skulle ikke vært med, vi hadde to iterasjoner, hvor litt for mye ble med.